src/styles/typography.tsx

```diff
@@ -73,4 +73,10 @@ export const typography = StyleSheet.create({
     fontSize: 12,
     letterSpacing: 0,
   },
+  // used for input in simple task
+  taskInput: {
+    fontFamily: fonts.sansSerif.regular,
+    fontSize: 20,
+    letterSpacing: 0,
+  },
 });
```

Comment:
I'm not sure if you should create separate topologies for one input. Maybe use an existing one or try to unify topologies for all inputs.

Reply:
Maybe we can change it later when we will se that there is another typo with the same props?

Reply:
Maybe we will change it later when find another typo with same props? For now I know only about this one.